plugins/content/config/model.schema

```diff
@@ -125,7 +125,7 @@
           "type": "string",
           "required": true,
           "default": "easeOutQuart",
-          "title": "Hide Easing",
```

Comment:
yes we have `easeOutQuart` for `_showEasing` and `easeInQuart` for `_hideEasing` [see config.json](https://github.com/adaptlearning/adapt_framework/blob/master/src/course/config.json#L11-L12). but mistakenly both titles was labeled as `Hide Easing` that's why I have corrected it!